code/datums/mood_events/_mood_event.dm

```diff
@@ -18,11 +18,11 @@
 	/// if false, it will be an overlay instead
 	var/special_screen_replace = TRUE
 	/// Owner of this mood event
-	var/mob/owner
+	var/mob/living/owner
 	/// List of required jobs for this mood event
 	var/list/required_job = list()
 
-/datum/mood_event/New(mob/M, ...)
+/datum/mood_event/New(mob/living/M, ...)
```

Comment:
If you would rename the argument from M to something more descriptive while you're at it. Thank you ^^

Reply:
Applied via https://github.com/tgstation/tgstation/pull/87747/commits/534fdd5248561a97469bd2da49ef616cf29fefbd